components/datetime/src/options/preferences.rs

```diff
@@ -33,12 +36,18 @@ use crate::fields;
 /// };
 /// ```
 #[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct Bag {
+    #[cfg_attr(
+        all(not(feature = "serialize_none"), feature = "serde"),
```

Comment:
Issue: There is no more serialize_none on main.  So you should feature-gate this only on the serde feature.

Reply:
Ah, I missed this one. I diffed against `main` and didn't see anymore that I missed.